com.unity.render-pipelines.universal/CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ All notable changes to this package will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [Unreleased]
+- Fixed null error when generating a new shader preview.
```

Comment:
This should go under the Fixed part in 14.0.0 :)

Reply:
Thanks!